src/components/ImageCardWithCTA/ImageCardWithCTA.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
```

Comment:
There is a lot of duplication here, understandably so, but from a React point of view we should try to make use of _composition_ - or in other words make use of props children to create a base component which has all of the common stuff and into which the markup which is different can be nested.

e.g.

```jsx
<CardBase>
  {description && <RichText>{description}</RichText>}
</CardBase>
```

This is an oversimplification but illustrates the basics. More than happy to chat through this if you like.

Longer term we want to overhaul how listings and cards are composed as it's pretty complex and therefore not ideal. Updating both this component and ImageCard to use the same CardBase will be good step in the right direction and certainly reduce duplication.

Some more info on composition:
https://reactjs.org/docs/composition-vs-inheritance.html
https://epicreact.dev/one-react-mistake-thats-slowing-you-down/